src/main/scala/cromwell/engine/CallActorState.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.engine
+
+sealed trait CallActorState {
+  def isTerminal: Boolean
+}
+
+case object CallNotStarted extends CallActorState {
```

Comment:
might as well make the extensions final in the ADT - not a big deal, there's a little blurb here: http://noelwelsh.com/programming/2015/06/04/more-on-sealed/

Also I'm not necessarily against it but curious on the thought process of bouncing these outside of the companion object. Is it just to declutter the companion?